data/entities.yaml

```diff
@@ -765,6 +765,15 @@
   capabilities: [grab, swap, give, place, atomic]
   properties: [portable]
 
+- name: binoculars
+  display:
+    attr: device
+    char: 'B'
+  description:
+  - Allows one to `scout` for other robots
```

Comment:
Personally, I would tend to go for a somewhat more whimsical/detailed description, but I don't think it matters all that much.

Reply:
Sure, there's room to revisit when inspiration strikes.